### ISSUE TITLE
[Bug] labelButtonProcessItem

### ISSUE THREAD

**Issue by danbars** [pqina/filepond#786]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you updated FilePond and its plugins?

- [X] I have updated FilePond and its plugins

### Describe the bug

Setting `labelButtonProcessItem` has no affect.
Maybe I misunderstand what this property is supposed to be doing.
My assumption was that it will allow me adding a label, in addition to the icon, for the upload button

I was trying to get to this result:
![image](https://user-images.githubusercontent.com/6115771/146328250-4827aea8-1d6c-4dd6-8337-0ee3815f34b7.png)

Is it possible and is this property the right way to do so?

### Reproduction

I'm using the vue plugin. Here's my code:
```
<file-pond
      accepted-file-types="image/jpeg, image/png, image/gif"
      name="file"
      :allow-multiple="false"
      :server="fileServerConfig"
      :allow-remove="false"
      :instant-upload="false"
      label-button-process-item="Upload"
 ></file-pond>
```

### Environment

```markdown
- Device: Macbook
- OS: Mac OS 11.6
- Browser: Chrome, 96
```

**Comment by rikschennink** [comment]
The text is there but it's hidden.

You can adjust the CSS to show it and then adjust the CSS on the parent <button> to position it the way you want.

<img width="371" alt="Screenshot 2021-12-17 at 08 06 43" src="https://user-images.githubusercontent.com/1132575/146503728-969da0c9-725e-41f8-8edc-24d7d52c0abf.png">

**Comment by danbars** [comment]
Thank for the explanation.
After playing with it, I was able to get the label located properly (almost), and it now looks like this:
![image](https://user-images.githubusercontent.com/6115771/146542898-407b09b7-a856-4124-8446-f7e77859b874.png)

However, When I hover the border is still a circle around the icon
![image](https://user-images.githubusercontent.com/6115771/146543102-fd26bbc5-9576-46fc-b0b5-e4a2f8f358bf.png)

When I tried to make the button itself larger, I got an ellipse and not a square or rounded square, and I couldn't find the reason for that.

Here's my css
```css
.filepond--file-action-button.filepond--action-process-item span {
  height: 100%;
  width: 56px;
  right: 25px;
  top: 2px;
  color: white;
  clip: unset;
  clip-path: unset;
  -webkit-clip-path: unset;
}
```

Thanks!
